.github/workflows/docker-image-build-push.yml

```diff
@@ -0,0 +1,31 @@
+# Build and push docker image
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - production
+      - master
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
@Charlie-Henry it'd be good to create a new docker token for this action and to rename `DOCKER_PASSWORD` to `DOCKER_TOKEN`. i can DM you the deets on acquiring that token.